src/vehicle.cpp

```diff
@@ -588,6 +588,11 @@ void vehicle::smash() {
     }
 }
 
+const std::string vehicle::disp_name()
```

Comment:
Btw. I meant to make the function const, not the return value, which is copied anyway and the const is essentially ignored here.